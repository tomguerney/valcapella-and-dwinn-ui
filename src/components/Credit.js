import React from 'react'
import Container from '@material-ui/core/Container'
import Typography from '@material-ui/core/Typography'

const CreditTypography = props => (
  <Container style={{ paddingTop: 5, paddingBottom: 10 }}>
    <Typography style={{ fontFamily: 'Alegreya Sans SC' }} align='center'>
      {props.role}
    </Typography>
    <Typography align='center'>{props.nameText}</Typography>
  </Container>
)

export default CreditTypography
