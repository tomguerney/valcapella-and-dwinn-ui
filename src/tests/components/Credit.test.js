import React from 'react'
import { shallow } from 'enzyme'
import Credit from '../../components/Credit'

test('should render Credit correctly', () => {
  const wrapper = shallow(<Credit role='Written and Narrated by' nameText='Thomas Benjamin Guerney'/>)
  expect(wrapper).toMatchSnapshot()
})