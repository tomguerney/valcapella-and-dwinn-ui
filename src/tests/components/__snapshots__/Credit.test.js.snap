// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Credit correctly 1`] = `
<WithStyles(ForwardRef(Container))
  style={
    Object {
      "paddingBottom": 10,
      "paddingTop": 5,
    }
  }
>
  <WithStyles(ForwardRef(Typography))
    align="center"
    style={
      Object {
        "fontFamily": "Alegreya Sans SC",
      }
    }
  >
    Written and Narrated by
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    align="center"
  >
    Thomas Benjamin Guerney
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Container))>
`;
